src/main/java/org/broadinstitute/hellbender/tools/copynumber/CollectAllelicCounts.java

```diff
@@ -112,56 +108,59 @@
             optional = false
     )
     protected File outputAllelicCountsFile;
-
-    @Argument(
-            doc = "Minimum mapping quality; reads with lower quality will be filtered out of pileup.",
-            fullName  = "minimumMappingQuality",
-            shortName = "minMQ",
-            optional = true
-    )
-    protected int minimumMappingQuality = 30;
+//
+//    @Argument(
+//            doc = "Validation stringency for all SAM/BAM/CRAM/SRA files read by this program.  The default stringency value SILENT " +
+//                    "can improve performance when processing a BAM file in which variable-length data (read, qualities, tags) " +
+//                    "do not otherwise need to be decoded.",
+//            fullName = StandardArgumentDefinitions.READ_VALIDATION_STRINGENCY_LONG_NAME,
+//            shortName = StandardArgumentDefinitions.READ_VALIDATION_STRINGENCY_SHORT_NAME,
+//            common = true,
+//            optional = true
+//    )
+//    protected ValidationStringency readValidationStringency = ReadConstants.DEFAULT_READ_VALIDATION_STRINGENCY;
```

Comment:
@cmnbroad Can you comment?  I think that if I leave it in I will get a conflicting name with another parameter that is already in LocusWalker.